### PR TITLE
[#172315782] Redirect tutorial articles to fastruby.io

### PR DESCRIPTION
**Pull request by nykka**
Pivotal Story Reference: https://www.pivotaltracker.com/story/show/172315782

This PR:
- Redirects those articles about Rails Upgrade to FastRuby.io.

To achieve this I've added a `redirect` variable into the from matter for those articles about Rails upgrades.
The variable: `redirect: https://www.fastruby.io/blog`
The articles:
_posts/2017-08-28-upgrade-to-rails-3.markdown
_posts/2017-10-16-upgrade-to-rails-3-1.md
 _posts/2017-11-07-upgrade-to-rails-3-2.md
 _posts/2017-11-08-upgrade-rails-from-3-2-to-4-0.markdown
 _posts/2017-11-24-upgrade-rails-from-4-0-to-4-1.markdown
_posts/2018-02-02-upgrade-rails-from-4-1-to-4-2.markdown
 _posts/2018-03-06-upgrade-rails-from-4-2-to-5-0.markdown
 _posts/2018-06-29-upgrade-rails-from-5-0-to-5-1.markdown
 _posts/2018-07-26-upgrade-rails-from-5-1-to-5-2.markdown

And then I've added a conditional into `<head>` to automatically redirect if the post has the `redirect` variable:
` {% if page.redirect %}<meta http-equiv="refresh" content="0; url={{ page.redirect }}{{ page.url }}">{% endif %}`

Additionally, I've added a text for those articules that iwll be redirected into the `post` layout to notice the user that is beeing redirected:
`<main class="content post-layout" role="main">
  <article class="post">
    <header class="post-header">
      <h1>Redirecting to FastRuby.io blog article</h1>
      <p>If you are not redirected automatically, please click <a href="{{ page.redirect }}{{ page.url }}">here</a></p>
    </header>
  </article>
</main>`

@ABizzinotto Let me know if you want to change this content.

### CODE REVIEW

**Comment by nykka** [comment]
I'll check the build errors tomorrow!

**Comment by nykka** [comment]
@ABizzinotto @arielj Ready for review

**Comment by arielj** [comment]
I'll leave the review to luciano, it looks good and straightforward but I'm not really familiar with the blog an jekyll

File: _layouts/default.html [hunk 1]
```diff
@@ -5,12 +5,16 @@
...
 
-  {%- capture page_title -%}{%- if page.category -%}{{page.category | titleize}} at {{site.description}}{%- elsif page -%}{{ page.title | titleize}} - {{ site.description }}{%- else -%}{{ site.description }} - {{ page.title | titleize }}{%- endif -%}{%- endcapture -%}
+  {%- capture page_title -%}{%- if page.category -%}{{page.category | titleize}} at
+  {{site.description}}{%- elsif page -%}{{ page.title | titleize}} - {{ site.description }}{%- else -%}{{ site.description }} - {{ page.title | titleize }}{%- endif -%}{%- endcapture -%}
 
   <title>{{ page_title }}</title>
 
-  {%- capture description -%}{%if page.meta.description -%}{{page.meta.description}}{%- elsif page.category -%}Articles about {{page.category | titleize}} in the {{site.description}}
-  blog.{%- elsif page -%}{{ page.excerpt | strip_html }}{%- else -%}{{ site.meta_description }}{%- endif -%}{%- endcapture -%}
+  {% if page.redirect %}
```
Comment (lubc) [inline _layouts/default.html:13]: @nykka I think you don't need a `redirect` variable. You can use the `canonical_url` that we already have. The articles that use that variable are the ones that were moved to fastruby (https://github.com/ombulabs/blog/pull/173), so we want to redirect all of those.

Comment (nykka) [reply _layouts/default.html:13]: Ok, I didn't know the articles moved to FastRuby were the ones with `canonical_url` variable. Thanks.

**Comment by lubc** [comment]
Discussion continued on Slack: https://ombulabs.slack.com/archives/CNEEYB68L/p1587568127002800

We can now close this PR. The work continued in https://github.com/ombulabs/blog/pull/423 and https://github.com/ombulabs/ombulabs.com/pull/189

**Pull request closed without merging**